README.md

```diff
@@ -136,7 +137,7 @@ navigate through folders.
 *   Create DICOM Store folder. If you try to create a folder at a different level, 
     you will get the error:  
     _mkdir: cannot create directory ‘new_folder’: Operation not permitted_
-*   mv - is not supported.
+*   mv - rename empty DICOM Store (previous DICOM Store will be deleted and a new one created).
```

Comment:
How about "previous DICOM store will be deleted and a new one will be created. Only supported for empty DICOM stores, otherwise an error will be returned"

Reply:
Done